Translations/bg.po

```diff
@@ -6666,26 +6705,30 @@ msgid ""
 " \n"
 "What? Oh, yes... uh, well, I suppose you could see what someone else knows."
 msgstr ""
+"Никога не съм се интересувал много от поезия. От време на време наемах странстващи музиканти, когато странноприемницата вървеше добре, но това вече изглежда много отдавна.\n"
```

Comment:
I'd put it... "Никога не съм изпитвал особен интерес към поезията."  instead of "Никога не съм се интересувал много от поезия.". Nevertheless, nitpicking here. Many ways to interpret.

Reply:
```suggestion
"Никога не съм се интересувал много от поезия. От време на време наемах странстващи музиканти, когато странноприемницата вървеше добре, но това вече изглежда много отдавна.\n"
```